libfwupd/fwupd-security-attr-private.h

```diff
@@ -14,13 +14,13 @@
 G_BEGIN_DECLS
 
 /**
- * FWUPD_SECURITY_ATTR_ID_ACPI_DMAR:
+ * FWUPD_SECURITY_ATTR_ID_PREBOOT_DMA_PROTECTION:
  *
  * Host Security ID attribute for ACPI DMAR table
  *
```

Comment:
The old name for this was org.fwupd.hsi.AcpiDmar etc.

Reply:
fixed